### ISSUE TITLE
Components idea

### ISSUE THREAD

**Issue by emil14** [emil14/frameword#1]
Create set of really useful css-agnostic web-components

They should be good in accessibility meaning, including good performance at all levels. They should be good from the API point of view

Then, when "core" (css+html) modules will be ready, HTML5 Web-Components should be created. After that, there should be wraps for React, Vue and Angular
